### ISSUE TITLE
only one instance of babel-polyfill is allowed

### ISSUE THREAD

**Issue by jgillich** [feathersjs/feathers-sequelize#3]
Apparently one can only require babel once: https://phabricator.babeljs.io/T1019

```
/home/jakob/devel/cloudio/node_modules/feathers-sequelize/node_modules/babel-polyfill/lib/index.js:8

  throw new Error("only one instance of babel-polyfill is allowed");
  ^

Error: only one instance of babel-polyfill is allowed
    at Object.<anonymous> (/home/jakob/devel/cloudio/node_modules/feathers-sequelize/node_modules/babel-polyfill/lib/index.js:8:9)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/home/jakob/devel/cloudio/node_modules/feathers-sequelize/lib/index.js:26:1)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
```

**Comment by daffl** [comment]
Dang, I didn't know that. We should probably conditionally include it for what we need in this case something like:

```js
if(typeof Object.assign !== 'function') {
  require('babel-polyfill');
}
```

**Comment by jgillich** [comment]
According to the comments, babel sets `global._babelPolyfill`
```
if (!global._babelPolyfill) {
  require('babel-polyfill')
}
```

**Comment by daffl** [comment]
Oh sweet. I should update it for the other adapters, too, later. If you have time would you mind submitting that change as pull request?

**Comment by jgillich** [comment]
Btw what is the status of feathers-waterline? :)

**Comment by daffl** [comment]
Thanks for the PR. It's released in v0.2.1. @ekryski started working on feathers-waterline maybe he can give a better status update than me.

**Comment by ekryski** [comment]
Ah :hankey:. I didn't know that either. I guess there are a couple places we probably need to change that.

@jgillich hoping to have waterline done in the next couple days. Just integrating our new feathers-errors updates. Will be a huge win.
